polling/processor.py

```diff
@@ -0,0 +1,51 @@
+"""
+Simplified polling processor which gets dispatches then waits to see if they're in progress
+"""
```

Comment:
It seems like this should be in a different folder because using it is mutually exclusive with using poller.py, but maybe I'm misunderstanding?

Reply:
I see how this is used in the tutorial... it would be nice to have the file make it clear how it differs or is a precursor to the other one, but unsure how to do so

Reply:
Yeah I can add a comment about this to the top. Should make it more clear.